src/routes/Home.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { styled } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import Divider from '@material-ui/core/Divider';
+import TextField from '@material-ui/core/TextField';
+import logo from '../assets/logo.png';
+import './Home.css';
+
+const ForgotPwdBtn = styled(Button)({
+  color: 'lightslategrey',
+  marginLeft: 'auto',
+  marginTop: '2px',
+  textTransform: 'none',
+  width: 'fit-content',
+});
+
+const TextFieldPwd = styled(TextField)({
+  marginTop: '8px',
+});
+
+const SigninBtn = styled(Button)({
```

Comment:
Can you change the hover color on the button. The text is hard to read when hovering over.